df.history.xml

```diff
@@ -243,11 +243,11 @@
             <stl-vector type-name='int32_t' name="unk_a0"/>
             <stl-vector type-name='int32_t' name="unk_b0"/>
             <stl-vector type-name='int32_t' name="unk_c0"/>
-            <int32_t name="unk_d0"/>
-            <int32_t name="unk_d4"/>
-            <int32_t name="unk_d8"/>
-            <int32_t name="unk_dc"/>
-            <stl-string name="unk_e0"/>
+            <int32_t name="original_histfig_id" ref-target='historical_figure' comment='ID of the historical figure who was reanimated to produce undead historical figure'/>
+            <int32_t name="original_race" ref-target='creature_raw' comment='race of the historical figure who was reanimated to produce undead historical figure'/>
+            <int32_t name="original_caste" ref-target='caste_raw' aux-value='$$.original_race' comment='caste of the historical figure who was reanimated to produce undead historical figure'/>
```

Comment:
Not sure about the aux-value; is this correct?

Reply:
Not sure. ref-target is the only one I really care about.